probe-rs/src/architecture/arm/communication_interface.rs

```diff
@@ -101,6 +101,7 @@ pub struct Initialized {
     dps: HashMap<DpAddress, DpState>,
     use_overrun_detect: bool,
     sequence: Arc<dyn ArmDebugSequence>,
+    current_tar: Option<u32>,
```

Comment:
Each Memory AP has its own `TAR` register, so I think this should be stored separately for each AP.